packages/oak/lib/core/App/index.js

```diff
@@ -32,7 +32,7 @@ export default forwardRef((options, ref) => {
 
   useEffect(() => {
     init();
-  }, [options]);
+  }, options.forceRender ? [options] : []);
```

Comment:
If `options` doesn't change it should not rerender unexpectedly. What was the use case/repro for this issue?